Dashboard/Server/SessionManagement/ServerSessionManager.cs

```diff
@@ -249,11 +263,43 @@ public void OnDataReceived(string serializedObject)
                     GetSummaryProcedure(deserializedObj);
                     return;
 
+                case "removeClient":
+                    RemoveClientProcedure(deserializedObj);
+                    return;
+
+                case "endMeet":
+                    return;

```

Comment:
Why is there no implementation here... shouldn't there be saveSummary and saveAnalytics sort of procedures here?

Reply:
Done